_inc/lib/core-api/class.jetpack-core-api-widgets-endpoints.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Widget information getter endpoint.
+ *
+ */
+class Jetpack_Core_API_Widget_Endpoint {
+
+	/**
+	 * @since 5.5.0
+	 *
+	 * @param WP_REST_Request $request {
+	 *     Array of parameters received by request.
+	 *
+	 *     @type string $id Widget id.
+	 * }
+	 *
+	 * @return WP_REST_Response|WP_Error A REST response if the request was served successfully, otherwise an error.
+	 */
+	public function process( $request ) {
+		$widgets = get_option( 'sidebars_widgets', array() );
```

Comment:
This var doesn't seem to be used anywhere

Reply:
Good catch, thank you!